api/context/context.go

```diff
@@ -236,3 +238,11 @@ func getSecurityTestConfig(securityTestName string) *types.SecurityTest {
 		TimeOutInSeconds: viper.GetInt(fmt.Sprintf("%s.timeOutInSeconds", securityTestName)),
 	}
 }
+
+func getMaxContainersAllowed() int {
+	maxContainersAllowed, err := strconv.Atoi(os.Getenv("HUSKYCI_DOCKERAPI_MAX_CONTAINERS_BEFORE_CLEANING"))
```

Comment:
I think it would be cool if we could allow the user to turn off this validation. Keeping all containers might be interesting to someone. What do you think, @Takehime?

Reply:
Good point, @Krlier! I have just created an issue (#252) so we can work on this soon.